src/components/input-dropdown/_base.scss

```diff
@@ -0,0 +1,21 @@
+.td-input-dropdown {
+    position: relative;
+}
+
+.td-input-dropdown__selection {
+    cursor: default;
+    pointer-events: none;
+}
```

Comment:
I wonder if it would be a good user experience if we made the input a trigger for the drop down list. So that if they tapped on the input, it would open up a list for selection.